lib/bn.js

```diff
@@ -885,13 +885,212 @@ BN.prototype._bigMulTo = function _bigMulTo(num, out) {
 
 BN.prototype.mulTo = function mulTo(num, out) {
   var res;
-  if (this.length + num.length < 63)
+  var len = this.length + num.length;
+  if (len < 63)
     res = this._smallMulTo(num, out);
-  else
+  else if (len < 1024)
     res = this._bigMulTo(num, out);
+  else
+    res = this._bigMulToF(num, out);
   return res;
 };
 
+
+// Private
+
+function FFTM(x, y) {
+  this.x = x;
+  this.y = y;
+}
+
+FFTM.prototype.makeRBT = function makeRBT(N) {
```

Comment:
1. It is the same every time the computation is performed, right?
2. Needs a comment ;